.github/workflows/rust.yml

```diff
@@ -30,6 +31,7 @@ jobs:
         uses: actions/checkout@v3
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
+        timeout-minutes: $CACHE_TIMEOUT_MINUTES
```

Comment:
I guess this has to be ${{ env.CACHE_TIMEOUT_MINUTES }}

Reply:
It was a bit more complex, this piece of technology is somehow typed so there's a need to `fromJSON` https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson
> Returns a JSON object or JSON data type for value. You can use this function to provide a JSON object as an evaluated expression or to convert environment variables from a string.

Thanks for looking into it anyway!